src/main/java/com/sopt/toss/service/ProductService.java

```diff
@@ -46,7 +48,24 @@ public BrandConDetailDto getBrandCon(long userId, Long productId) {
                 .orElseThrow(() -> new NotFoundException(NOT_FOUND_USER_EXCEPTION, NOT_FOUND_USER_EXCEPTION.getMessage()));
         Product product = productRepository.findById(productId)
                 .orElseThrow(() -> new NotFoundException(NOT_FOUND_PRODUCT_EXCEPTION, NOT_FOUND_PRODUCT_EXCEPTION.getMessage()));
-        Like like = likeRepository.findByUserAndProduct(user, product);
-        return BrandConDetailDto.toDto(product, like.isLike());
+        boolean isLike;
+        Like like = likeRepository.findByUserAndProduct(user, product).orElse(null);
+        if(like ==null) isLike = false;
+        else isLike = like.isLike();
```

Comment:
```suggestion
        Like like = likeRepository.findByUserAndProduct(user, product).orElse(null);
        boolean isLike = like != null && like.isLike();
```
별 차이는 없지만 요런식은 어떤가요?!

Reply:
와 논리연산자는 생각 못했는데 훨씬 깔끔하네여 바로 수정-*